coconut/cmd/configuration_history.go

```diff
@@ -0,0 +1,48 @@
+/*
+ * === This file is part of ALICE O² ===
+ *
+ * Copyright 2018 CERN and copyright holders of ALICE O².
```

Comment:
Please make sure that the year is correct in all headers.

Reply:
👍